plentyofpets/lib/screens/add_edit_pet_screen.dart

```diff
@@ -0,0 +1,188 @@
+                      labelStyle: TextStyle(
+                        fontSize: 15,
+                      ),
+                      contentPadding: EdgeInsets.symmetric(horizontal: 110),
+                    ),   
+                    allowClear: true,
+                    //hint: const Text('Use the dropdown to select adoption status'),
+                    items: availableOptions.map((availability)=> DropdownMenuItem(
+                      value: availability,
+                      child: Text(availability),)).toList(),
+                  ),
+                  const SizedBox(height: 20),
+ 
+                  FormBuilderFilterChip(
+                    name: 'Disposition',
+                    decoration: const InputDecoration(
+                      labelText: 'Disposition',
+                      labelStyle: TextStyle(
+                        color: Colors.black,
+                        fontSize: 20,
+                        fontWeight: FontWeight.bold
+                      ),
+                    contentPadding: EdgeInsets.symmetric(horizontal: 25),
+                    ),    
+                    options: const [
+                      FormBuilderFieldOption(value:'Kids', child: Text('Good with Kids')),
+                      FormBuilderFieldOption(value: 'other animals', child: Text('Good with other animals')),
+                      FormBuilderFieldOption(value: 'leashed', child: Text('Must be leashed')),
+                      FormBuilderFieldOption(value: 'House-Trained', child: Text('House-Trained')),
+                      FormBuilderFieldOption(value: 'Exercise', child: Text('Needs regular exercise')),
+                      FormBuilderFieldOption(value: 'Calm', child: Text('Calm')),
+                    ],
+                  ),
+                  const SizedBox(height: 20),
+                  
+                  FormBuilderTextField(
+                    name: 'Breed' ,
+                    decoration: const InputDecoration(
+                      labelText: 'Breed',
+                      labelStyle: TextStyle(
+                        fontSize: 15,
+                      ),
+                      contentPadding: EdgeInsets.symmetric(horizontal: 142),
+                    ),
+                  ),
+                  const SizedBox(height: 20),
+                  
+                  FormBuilderTextField(
+                    name: 'Location' ,
+                    decoration: const InputDecoration(
+                      labelText: 'Location',
+                      labelStyle: TextStyle(
+                        fontSize: 15,
+                      ),
+                      contentPadding: EdgeInsets.symmetric(horizontal: 135),
+                    ),
+                  ),
+                  const SizedBox(height: 20),
+                  
+                  FormBuilderTextField(
+                    name: 'Pet Name' ,
+                    decoration: const InputDecoration(
+                      labelText: 'Enter Pet Name',
```

Comment:
Maybe for consistency, consider changing this to just 'Pet Name'? None of the other fields say 'Enter' right now.

Reply:
Sounds good. It's been changed in the new commit